core/flyout_base.js

```diff
@@ -536,6 +536,22 @@ Blockly.Flyout.prototype.show = function(xmlList) {
         } else {
           gaps.push(default_gap);
         }
+      } else if ((tagName == 'BUTTON') &&
+        (xml.getAttribute('type') === 'status')) {
```

Comment:
The scratch-blocks convention is to use `==`

Reply:
Also does this need to be on two separate lines? If so, you need to indent an extra 2 spaces on this second line.

Reply:
👍